.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Trigger tests in corporate
+      run: |
+        curl --request POST \
+        --url https://circleci.com/api/v1.1/project/github/${{ secrets.CORPORATE_REPO }}/tree/master \
+        --user '${{ secrets.CIRCLECI_USER }}' \
```

Comment:
In Zapier, I believe I had to specify my API token as well, as it's a private repo.

Reply:
I took that one from zapier :) it works with `--user` (tested it locally)

Reply:
Maybe rename CIRCLECI_USER to CIRCLECI_USER_TOKEN to be most clear here